src/modules/python/pbs/v1/_base_types.py

```diff
@@ -532,6 +532,9 @@ def __sub__(self,other):
 	# return would look in here for comparison operators
 	# and not in _size's richcompare.
         return size(s.__sub__(o))
+    
+    def __deepcopy__(self, mem):
```

Comment:
@minghui-liu However code looks good but curious to know what will be benefits of using ```copy.deepcopy()``` over directly ```pbs.size()``` to create copy ?

Reply:
`deepcopy` recursively creates copies of members within a type. So if some other types have `pbs.size` as member and want to use deepcopy, then `pbs.size` needs to support it.

Reply:
Ah I see... Thanks for explanation... Now thinking more, should our all PBS types support __deepcopy__?
(Off-course not in this PR 😄)

Reply:
Yes I think most types can benefit from deepcopy. Maybe except some types that are designed to have only one instance.

Reply:
@hirenvadalia : Yes, I think all types that could be referenced in a deepcopy() operation should be made to support it. Otherwise, the implementation would be incomplete, IMHO.